src/commands/force/user/password/generate.ts

```diff
@@ -33,12 +33,7 @@ export class UserPasswordGenerateCommand extends SfdxCommand {
   private passwordData: PasswordData[] = [];
 
   public async run(): Promise<PasswordData[]> {
-    if (this.flags.onbehalfof) {
-      // trim the usernames to avoid whitespace
-      this.usernames = this.flags.onbehalfof.map((user) => user.trim());
```

Comment:
Does the whitespace not matter?

Reply:
flags now handles that: https://github.com/forcedotcom/cli-packages/pull/138